docs/install.md

```diff
@@ -79,16 +79,14 @@ This will deploy the latest version of `gef`'s _main_ branch from Github. If no
 To contribute to GEF, you might prefer using git directly.
 
 ```bash
-$ git clone https://github.com/hugsy/gef.git
+$ git clone --branch main https://github.com/hugsy/gef.git
 $ echo source `pwd`/gef/gef.py >> ~/.gdbinit
 ```
 
-GEF is in very active development, so the default branch is `dev`. This is the branch you must use if you intend to submit pull requests.
-
-However if you prefer a more stable life, you can then switch to the `main` branch:
+GEF is in very active development, if you want enjoy the last new features or submit pull requests you must switch to the `dev` branch:
```

Comment:
I don't really agree to that, we'll receive PR against `main`. This section is for contribution, they should use `dev`